src/ReTestItems.jl

```diff
@@ -865,6 +868,13 @@ function runtestitem(
             push!(body.args, :(const $setup = $ts_mod))
         end
         @debugv 1 "Setup for test item $(repr(name)) done$(_on_worker())."
+
+        # add the `test_end_expr` to a module to be run after the test item
+        test_end_body = copy(body)
+        test_end_mod_expr = :(module $(gensym(name * " test_end")) end)
+        append!(test_end_body.args, test_end_expr.args)
+        test_end_mod_expr.args[3] = test_end_body
```

Comment:
should we also `softscope` the args, like we do for the `@testitem` body (but don't for the `worker_init_expr`)?

Reply:
That seems reasonable to me, yeah

Reply:
done